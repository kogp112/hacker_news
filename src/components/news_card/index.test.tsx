import * as React from "react";
import * as renderer from "react-test-renderer";
import NewsCard from "./index";

let container = null;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  container = null;
});

it("should render news card", () => {
  const newsId = "";
  const index = 0;

  const component = renderer
  .create(<NewsCard newsId={newsId} index={index} />)
  .toJSON();
  expect(component).toMatchSnapshot();
});