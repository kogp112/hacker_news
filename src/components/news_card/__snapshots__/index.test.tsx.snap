// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render news card 1`] = `
<div
  className="card_0"
>
  <div
    onClick={[Function]}
  >
    <p
      className="container"
    />
  </div>
</div>
`;
